components/apprentice-status-indicator/apprentice-status-indicator.module.scss

```diff
@@ -2,6 +2,28 @@
 @use '../../styles/spacing' as *;
 @use '../../styles/fonts' as *;
 
+// This mixin should be moved into a mixin partial:
+@mixin visually-hidden {
+  position: fixed;
+  right: 100%;
+  bottom: 100%;
+  clip: rect(1px, 1px, 1px, 1px);
+  clip-path: inset(50%);
+
+  &:focus {
+    left: 0;
+    right: auto;
+    top: 0;
+    bottom: auto;
+    clip: auto;
+    clip-path: none;
+  }
+}
+
+.status-label {
+  @include visually-hidden;
+}
```

Comment:
Just making a note: Let's be sure to coordinate this merge with #31 since it includes the extracted mixin